libs/dh/shared/data-access-msw/src/lib/browser.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { setupWorker } from 'msw';
+import { handlers, onUnhandledRequest } from './handlers';
+
+try {
+  const worker = setupWorker(...handlers);
+  worker.start({ onUnhandledRequest });
```

Comment:
suggestion: If the code is exported inside a function, the `try/catch` will probably not be needed.

Reply:
Unfortunately I'm pretty sure it is. Problem is that it otherwise will fail in our tests because api.module imports the environment